<a href="https://colab.research.google.com/github/IvanHernandez26/Trabajo-Analisis-de-Datos/blob/main/Jueveslab_23_10_2025.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [2]:
import pandas as pd
alumnos = pd.read_csv('/content/datasets/alumnos.csv')
alumnos

Alumnos     Genero  Edad  Promedio
0    Braulio  Masculino    19        77
1     Mihael  Masculino    19        82
2     Aseret   Femenino    19        85
3       Aldo  Masculino    22        80
4    Zabdiel  Masculino    18        88
5       Ivan  Masculino    19        72
6   Vannessa   Femenino    19        73
7       Gaby   Femenino    17        71
8       Misa  Masculino    19        75
9       Alan  Masculino    19        70
10    Alexis  Masculino    19        90
11    Brayan  Masculino    19        87
12    Diego   Masculino    19        85
13    Efren   Masculino    20        78
14      Luis  Masculino    20        80
15     Yamil  Masculino    22        82

In [4]:
alumnos['Edad'].describe()
#Numerica

count    16.0000
mean     19.3125
std       1.2500
min      17.0000
25%      19.0000
50%      19.0000
75%      19.2500
max      22.0000
Name: Edad, dtype: float64

In [5]:
alumnos["Genero"].describe()
#Categorica

count            16
unique            2
top       Masculino
freq             13
Name: Genero, dtype: object

In [7]:
media_Edad = alumnos['Edad'].mean()
alumnos["Edad"].map(lambda Edad: Edad - media_Edad)
#Mapear Abajo

0    -0.3125
1    -0.3125
2    -0.3125
3     2.6875
4    -1.3125
5    -0.3125
6    -0.3125
7    -2.3125
8    -0.3125
9    -0.3125
10   -0.3125
11   -0.3125
12   -0.3125
13    0.6875
14    0.6875
15    2.6875
Name: Edad, dtype: float64

In [8]:
media_Edad = alumnos['Edad'].mean()
alumnos["Edad"].map(lambda Edad: Edad - media_Edad)
#Mapear Arriba

0    -0.3125
1    -0.3125
2    -0.3125
3     2.6875
4    -1.3125
5    -0.3125
6    -0.3125
7    -2.3125
8    -0.3125
9    -0.3125
10   -0.3125
11   -0.3125
12   -0.3125
13    0.6875
14    0.6875
15    2.6875
Name: Edad, dtype: float64

El dataset incluye datos de 16 estudiantes, nombre, genero, edad y promedio de calificaciones. La edad es una variable numerica con un promedio de 19.31 años, oscilando entre 17 y 22. El genero es categorico con dos valores, Masculino y Femenino, predominando el masculino 13 de 16. Se calculo la diferencia entre cada edad y el promedio
